### ISSUE TITLE
When I enable authorization, how can options in the command line be transferred in the grpc interface parameters in v3.4?

### ISSUE THREAD

**Issue by AI1186780944** [etcd-io/etcd#13033]
eg:When I enable authorization, My command line <etcdctl role list --user="root:123456"> can be executed normally through etcdctl. So when I call with the grpc interface, how to pass the parameters of the following option into the interface?If I don't normally pass in the optional parameter <--user="root:123456">, there will be a "user name is empty" error.

**Comment by mitake** [comment]
Username and password are used as parameters of `Authenticate()` method. The method returns a token and the token will be attached to gRPC connection as its metadata like this: https://github.com/etcd-io/etcd/blob/main/client/v3/client.go#L268

**Comment by AI1186780944** [comment]
yes. I did the same thing. Before, I didn't know the rules of grpc using token, which caused the problem. The problem has been solved. Thank you very much.At the same time, can I make a suggestion. Can v3.5 provide such an interface as "query the list of all historical versions of a key"? What I see now is that it only provides the interface of "querying the details of a certain version of a certain key". The reason is that if I don't know the versions of the key in advance, how can I query the detailed data of the corresponding versions of the key?
